postgres_stats.c

```diff
@@ -242,27 +242,39 @@ static void merge_stats(pg_stat_list *pg_stats, proc_stat_list proc_stats)
 
 static void read_proc_cmdline(pg_stat *stat)
```

Comment:
Could you please add a comment about expected format for the `cmdline` file contents?  It would make easier to make sense of the code here.